components/Tabs/Tabs.js

```diff
@@ -2,50 +2,59 @@
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
+    this.element = element;
     
     // Get the custom data attribute on the Link
-    // this.data;
-    
+    this.data = document.querySelector(`.tabs-link[data-tab='${this.element.dataset.tab}']`);
```

Comment:
Another way to get the custom data attribute is by using
   * this.data = this.element.dataset.tab;*